WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController+QuickStart.swift

```diff
@@ -13,15 +13,18 @@ extension MySiteViewController {
             return
         }
 
+        switch element {
+        case .siteIcon, .siteTitle, .viewSite, .pages, .stats, .mediaScreen:
```

Comment:
I believe .sharing should also be added here, since it's accessed via the new More item

Reply:
Updated, thanks!